src/common/libkvs/kvs_txn.c

```diff
@@ -332,7 +318,7 @@ int flux_kvs_txn_unlink (flux_kvs_txn_t *txn, int flags,
     }
     if (validate_flags (flags, 0) < 0)
         goto error;
-    if (flux_kvs_txn_put_treeobj (txn, flags, key, NULL) < 0)
+    if (flux_kvs_txn_put_treeobj (txn, flags, key, json_null ()) < 0)
```

Comment:
I think we'll mem-leak here, because ```txn_encode_op``` packs using ```s:O``` (that's upper case O).  I think we need to save a reference to the object returned by ```json_null ()``` and decref it.

Reply:
The jansson manual says json_null() is special, so I think the json_decref would be a no-op but it's good form at least.  I'll go ahead and add it.